### PR TITLE
Documentation to explain the recently added `pages` feature

### PR DESCRIPTION
**Pull request by JoeChapman**
Docs for pages

### CODE REVIEW

File: source/documentation/basic-customisation.md [hunk 1]
```diff
@@ -13,6 +13,59 @@ apps/my-hof-form/
...
+},
+pages: {
+  '/cookies': 'cookies',
+  '/privacy-policy': 'privacy',
+}
+```
+
+### Create the views
+
+By default views are loaded from the views directory in the current working directory. If necessary, views can be loaded from a custom location, set by adding a `views` option to the configuration for the route, e.g;
```
Comment (lennym) [inline source/documentation/basic-customisation.md]: I think this gives the impression that the custom views directory will *only* be for static pages. If we want to document a custom views location, then I think it should not be in this particular context.

File: source/documentation/basic-customisation.md [hunk 2]
```diff
@@ -13,6 +13,59 @@ apps/my-hof-form/
...
+views: './my/custom/views',
+pages: {
+  '/cookies': 'cookies',
+  '/privacy-policy': 'privacy',
+}
+```
+
+Make sure you have a view with the appropriate name in the apps\` views directory, one called `cookies.html` and another called `privacy-policy.html`.
+
+Now if you restart your server (`Ctrl+C` to stop, then run the same command again) and go to `/cookies` in the browser you should see the content of the cookies view rendered to the browser.
```
Comment (lennym) [inline source/documentation/basic-customisation.md]: ~We've explained how to restart the server earlier in the guide, so I don't think it needs repeating.~

Apparently not. You've put this section earlier than I thought.

**Comment by lennym** [comment]
I don't think I'd put this content so early in the guide. Where it is it's literally the first thing someone comes to after they've created a form, which could lead to the impression that it's necessary.

I'd put it after all of the actual _form_ related content. Possibly after "Content and i18n".

**Comment by JoeChapman** [comment]
@lennym now moved to more suitable location

**Pull request merged**